rfcs/text/0044-add-apple-platform-specific-fields.md

```diff
@@ -0,0 +1,119 @@
+# 0000: Name of RFC
+<!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
+
+- Stage: **0 (strawperson)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **TBD** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+
+
+### Summary
+This RFC proposes the addition of Apple platform-specific fields to the ECS schema. This enhancement will enable security software vendors to more accurately map out data, particularly for Apple platforms.
+
+The following feelds needs to be considered being added:
+
+## Fields
+
+##### Proposed New Fields for Process object
+
+Field | Type | Example | Description
+--- | --- | --- | ---
+responsible	| keyword	| Terminal.app	| The responsible process on macOS, from an ancestry perspective, is the process that originally launched or spawned a given process.
+platform_binary	| boolean	| true	| Indicates wethether this process executable is a default platform binary shipped with the operating system.
+endpoint_security_client	| boolean	| true	| Indicates wethether this process executable is an Endpoint Security client.
+
+##### Proposed New Fields for Code Signature object
+
+Field | Type | Example | Description
+--- | --- | --- | ---
+flags	| string	| 570522385	| The flags used to sign the process.
+
+##### Proposed New Fields for Hash object
+
+Field | Type | Example | Description
+--- | --- | --- | ---
+cdhash	| keyword	| 3783b4052fd474dbe30676b45c329e7a6d44acd9	| The Code Directory (CD) hash of an executable
+
+##### Proposed New Fields for Host object
+
+Field | Type | Example | Description
+--- | --- | --- | ---
+serial_number	| keyword	| DJGAQS4CW5	| The unique serial number serves as a distinct identifier for each device, aiding in inventory management and device authentication.
```

Comment:
I'm still wondering if it's not better fit for https://www.elastic.co/guide/en/ecs/current/ecs-device.html?

Reply:
even short description shows it's about device `short: Serial Number of the device`